apps/aehttpclient/lib/client.ex

```diff
@@ -115,16 +130,11 @@ defmodule Aehttpclient.Client do
     end
   end
 
-  def json_response(body) do
-    response = Poison.decode!(body)
-    {:ok, response}
-  end
-
   defp send_to_peer(data, uri) do
-    HTTPoison.post uri, Poison.encode!(data),
-      [{"Content-Type", "application/json"}]
+    HTTPoison.post(uri, Poison.encode!(data), [{"Content-Type", "application/json"}])
   end
 
+  # TODO: what is this function even doing?
   defp get_local_port() do
```

Comment:
What is going on in this function? The code here seems ridiculous, and either needs a comment explaining its operations, or can be refactored to pattern match against array internals

Reply:
Its a little weird, we need to get the port we are listening on